components/dashboard/src/settings/Teams.tsx

```diff
@@ -527,6 +528,16 @@ function AllTeams() {
                 <div className="flex-grow ">
                     <h3 className="self-center">All Teams</h3>
                     <h2>Manage teams and team members.</h2>
+                    <ItemsList className="pt-2">
+                        <Item className="w-full bg-yellow-200 hover:bg-yellow-200 py-6 px-6">
```

Comment:
This is also using some new colors for this alert component variant, compared to the component used for Theia deprecation or garbage collection. 🖍️